.github/workflows/build.yml

```diff
@@ -0,0 +1,47 @@
+name: MicroProfile Config Build
+
+on:
+  push:
+    paths-ignore:
+      - 'editorconfig'
+      - '.gitattributes'
+      - '.gitignore'
+      - 'CONTRIBUTING*'
+      - 'CODEOWNERS'
+      - 'KEYS'
+      - 'LICENSE'
+      - 'NOTICE'
+      - 'README*'
+      - 'site.yaml'
+  pull_request:
+    paths-ignore:
+      - 'editorconfig'
+      - '.gitattributes'
+      - '.gitignore'
+      - 'CONTRIBUTING*'
+      - 'CODEOWNERS'
+      - 'KEYS'
+      - 'LICENSE'
+      - 'NOTICE'
+      - 'README*'
+      - 'site.yaml'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [8, 11]
+    name: build with jdk ${{matrix.java}}
```

Comment:
@radcortez at the moment, we are building all MP API Jars with Java 8. With the matrix including 11, which Java version will the Github action use by default? I am new to the action and this might be a naive question.

Reply:
In this particular case, there is no default, it will just run 2 separate builds, one with JDK 8 and another with 11. I think this might be useful, so we can be up to date to any things that might break on either version. It generate 2 separate status check, but we can setup to just pass the JDK 8 build and we keep the information about 11. Both are run on push or pull request events.

This does not handle the release of the project. For that, we need to add a separate workflow. In that case, we won't use a matrix strategy and just set the JDK to 8.